plugins/woocommerce-blocks/assets/js/blocks/checkout/edit.tsx

```diff
@@ -86,8 +86,8 @@ export const Edit = ( {
 	}, [ clientId ] );
 
 	const defaultTemplate = [
-		[ 'woocommerce/checkout-fields-block', {}, [] ],
 		[ 'woocommerce/checkout-totals-block', {}, [] ],
+		[ 'woocommerce/checkout-fields-block', {}, [] ],
```

Comment:
This swapping of position doesn't impact the `save` function, therefore, WP doesn't identify any invalid content. This means that creating a deprecation is pointless in this case. (I have tried it)

Reply:
Visually speaking, those would be now inverted in the editor list, not sure if that's reintroducing the issue again but on desktop instead of mobile?

<img width="353" alt="image" src="https://github.com/user-attachments/assets/62e25422-a037-43ab-b3ca-3d9ad38481e4">